haskell/src/SwiftNav/SBP/Encoding.hs

```diff
@@ -0,0 +1,33 @@
+-- |
+-- Module:      SwiftNav.SBP.Encoding
+-- Copyright:   Copyright (C) 2015 Swift Navigation, Inc.
+-- License:     LGPL-3
+-- Maintainer:  Mark Fine <dev@swiftnav.com>
+-- Stability:   experimental
+-- Portability: portable
+--
+-- Encoding utilities.
+
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
Not sure. Perhaps we don't want warnings like this disabled across the entire project?